README.md

```diff
@@ -8,7 +8,7 @@ This repo outlines **a format for APIs, libraries and datasets containing and wo
 
 It tries to keep things simple:
 
-- JSON and [ndjson](http://ndjson.org) only
+- JSON and [ndjson]([http://ndjson.org](https://github.com/ndjson/ndjson-spec)) only
```

Comment:
It should be `[ndjson](https://github.com/ndjson/ndjson-spec)`, right?

Reply:
Sorry, copy-paste failure on GH web code editor 😓  Fixed! Remember to squash, as I didn't amend this one. Just a drive-by from github web UI, for this time.